src/main/java/org/edumips64/core/cache/CacheLine.java

```diff
@@ -0,0 +1,16 @@
+package org.edumips64.core.cache;
+
+// A cache line represents a block in the cache.
+public class CacheLine {
```

Comment:
Does this class need to be public? Or can it have package visibility? That would make the `public` fields more appropriate.